.gitignore

```diff
@@ -1,6 +1,5 @@
 # Django files
 edmontonpy/*.sqlite3
-edmontonpy/settings/local_settings.py
```

Comment:
Lets keep this line, I don't want people accidentally adding their local_settings.py files to git.

Reply:
I think we will be adding to the local_settings.py file in the future for some default settings to get people up and running in a reasonable amount of time.

Though I agree with you with accidental commits of local_settings.py so we'll keep it in there.

Thank you for the review 👍